python/interpret_community/captum/explanation.py

```diff
@@ -0,0 +1,53 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+"""Defines the interface to create an interpret-community style explanation from Captum."""
+
+import numpy as np
+
+from ..explanation.explanation import _create_local_explanation, _create_global_explanation, \
+    _aggregate_global_from_local_explanation, _aggregate_streamed_local_explanations
+from ..common.constants import ExplainParams, ExplainType, Defaults, ModelTask
+
+
+class CaptumAdapter(object):
```

Comment:
maybe instead of CaptumAdapter this should just be a more general ExplanationAdapter since I think it will work with SHAP and LIME too